### PR TITLE
Move win x64 builds from AppVeyor to TravisCI

### PR DESCRIPTION
**Pull request by Mikhus**

### CODE REVIEW

File: appveyor.yml [hunk 1]
```diff
@@ -4,72 +4,69 @@ environment:
...
+  - {"nodejs_version": "7", "npm_version": "4"}
+  - {"nodejs_version": "8", "npm_version": "4"}
+  - {"nodejs_version": "9", "npm_version": "5"}
+  - {"nodejs_version": "10", "npm_version": "6"}
+  - {"nodejs_version": "11", "npm_version": "6"}
 
 platform:
-  - x86
-  - x64
+- x86
```
Comment (Mikhus) [inline appveyor.yml]: Can't get a clue here how to keep node6,7 x64, skipping all other versions for x64...
@royaltm any ideas?

Comment (royaltm) [reply appveyor.yml]: https://www.appveyor.com/blog/2018/04/25/specialized-build-matrix-configuration-in-appveyor/

**Comment by Mikhus** [comment]
I guess I've tried all possible combinations with '-', without, etc, nothing works :)

**Comment by royaltm** [comment]
I don't know if it helps but I see that you sometimes indent arrays below map keys and sometimes not:
```
for:
-  <---- here is not indented
  environment:
    matrix:
      only:
        <---- here is indented
        - {"nodejs_version": "6", "npm_version": "3"}
        - {"nodejs_version": "7", "npm_version": "4"}
```

**Comment by Mikhus** [comment]
> I don't know if it helps but I see that you sometimes indent arrays below map keys and sometimes not:
> 
> ```
> for:
> -  <---- here is not indented
>   environment:
>     matrix:
>       only:
>         <---- here is indented
>         - {"nodejs_version": "6", "npm_version": "3"}
>         - {"nodejs_version": "7", "npm_version": "4"}
> ```

That's what I've tried initially and it does not work. Then I just started to try different combinations with identation moving sections, etc...
This variant just gives an error:
`Error parsing appveyor.yml: Environment variables "matrix" section must be a sequence of mappings. (Line: 24, Column: 7)`

**Comment by Mikhus** [comment]
> Have you tried?
> 
> ```
> for:
>   -
>     matrix:
>       only:
>         - {"nodejs_version": "6", "npm_version": "3"}
>         - {"nodejs_version": "7", "npm_version": "4"}
>     platform:
>       - x64
>       - x86
> ```

Yes, it gives no error, but the configuration is missing x64, so it does nothing:

```
Environment: nodejs_version=6, npm_version=3
Environment: nodejs_version=7, npm_version=4
Environment: nodejs_version=8, npm_version=4
Environment: nodejs_version=9, npm_version=5
Environment: nodejs_version=10, npm_version=6
Environment: nodejs_version=11, npm_version=6
```

**Comment by royaltm** [comment]
maybe we cant specify platform here as an array

**Comment by royaltm** [comment]
```
for:
  -
    matrix:
      only:
        - {"nodejs_version": "6", "npm_version": "3"}
        - {"nodejs_version": "7", "npm_version": "4"}
    platform: x64
  -
    matrix:
      only:
        - {"nodejs_version": "6", "npm_version": "3"}
        - {"nodejs_version": "7", "npm_version": "4"}
    platform: x86
```

**Comment by Mikhus** [comment]
> for: - matrix: only: - {"nodejs_version": "6", "npm_version": "3"} - {"nodejs_version": "7", "npm_version": "4"} platform: x64 - matrix: only: - {"nodejs_version": "6", "npm_version": "3"} - {"nodejs_version": "7", "npm_version": "4"} platform: x86

does not work, same result - no error, but no x64 builds in the list

**Comment by royaltm** [comment]
Hold on, maybe this is wrong approach.
https://www.appveyor.com/docs/build-configuration/#exclude-configuration-from-the-matrix

**Comment by royaltm** [comment]
What about:

```
platform:
  - x86
  - x64

matrix:
  exclude:
    - platform: x64
      - {"nodejs_version": "8", "npm_version": "4"}
      - {"nodejs_version": "9", "npm_version": "5"}
      - {"nodejs_version": "10", "npm_version": "6"}
      - {"nodejs_version": "11", "npm_version": "6"}
```

**Comment by royaltm** [comment]
> What about:
> 
> ```
> platform:
>   - x86
>   - x64
> 
> matrix:
>   exclude:
>     - platform: x64
>       - {"nodejs_version": "8", "npm_version": "4"}
>       - {"nodejs_version": "9", "npm_version": "5"}
>       - {"nodejs_version": "10", "npm_version": "6"}
>       - {"nodejs_version": "11", "npm_version": "6"}
> ```

This will not work, but maybe this:


```
platform:
  - x86
  - x64

matrix:
  exclude:
    - {"platform": "x64", "nodejs_version": "8", "npm_version": "4"}
    - {"platform": "x64", "nodejs_version": "9", "npm_version": "5"}
    - {"platform": "x64", "nodejs_version": "10", "npm_version": "6"}
    - {"platform": "x64", "nodejs_version": "11", "npm_version": "6"}
```

**Comment by Mikhus** [comment]
> > What about:
> > ```
> > platform:
> >   - x86
> >   - x64
> > 
> > matrix:
> >   exclude:
> >     - platform: x64
> >       - {"nodejs_version": "8", "npm_version": "4"}
> >       - {"nodejs_version": "9", "npm_version": "5"}
> >       - {"nodejs_version": "10", "npm_version": "6"}
> >       - {"nodejs_version": "11", "npm_version": "6"}
> > ```
> 
> This will not work, but maybe this:
> 
> ```
> platform:
>   - x86
>   - x64
> 
> matrix:
>   exclude:
>     - {"platform": "x64", "nodejs_version": "8", "npm_version": "4"}
>     - {"platform": "x64", "nodejs_version": "9", "npm_version": "5"}
>     - {"platform": "x64", "nodejs_version": "10", "npm_version": "6"}
>     - {"platform": "x64", "nodejs_version": "11", "npm_version": "6"}
> ```

Holly shit! It works! :)))

**Comment by royaltm** [comment]
Now I'am lost :)

**Comment by royaltm** [comment]
I must admit that this `for` expression isn't very intuitive.

**Comment by Mikhus** [comment]
Yeah, agree on that...

**Comment by Mikhus** [comment]
This one was also given the list of expected versions, but there was no clarification about platform on build page:

~~~
for:
  -
    platform: x64

    environment:
      matrix:
        - {"nodejs_version": "6", "npm_version": "3"}
        - {"nodejs_version": "7", "npm_version": "4"}
~~~

But it possibly might work as expected... who knows...

**Comment by royaltm** [comment]
@Mikhus Thank you for your contribution.

**Pull request merged**